config/initializers/assets.rb

```diff
@@ -3,6 +3,9 @@
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
 
+Rails.application.config.assets.precompile += %w( papers.css )
+Rails.application.config.assets.precompile += %w( authors.css )
```

Comment:
Lines 6 and 7 should be combined: `Rails.application.config.assets.precompile += %w( papers.css authors.css )